src/form/acr.jl

```diff
@@ -562,3 +731,34 @@ function constraint_mc_transformer_power_dy(pm::_PM.AbstractACRModel, nw::Int, t
         )
     end
 end
+
+
+""
+function constraint_mc_storage_losses(pm::_PM.AbstractACRModel, i::Int; nw::Int=pm.cnw, kwargs...)
```

Comment:
The `constraint_storage_losses` in PowerModels is already generalized to the MC case.  So you I would put this implementation in PowerModels.

Reply:
I would also expect that the constraint template for `constraint_mc_storage_losses` dispatches to the PM's `constraint_storage_losses` in all cases.

Reply:
That makes sense that it should go into PowerModels instead. In the short term, in order to get to our releases on schedule would you suggest we attempt to quickly add this to PowerModels, or make an Issue here and in PowerModels, and merge this function as is?